_posts/en/newsletters/2021-06-09-newsletter.md

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Bitcoin Optech Newsletter #152'
+permalink: /en/newsletters/2021/06/09/
+name: 2021-06-09-newsletter
+slug: 2021-06-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to allow LN nodes to receive
+payments without keeping their private keys online all the time.  Also
+included are our regular sections with the summary of a Bitcoin Core PR
+Review Club meeting, announcements of new software releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Receiving LN payments with a mostly offline private key:** in 2019,
+  developer ZmnSCPxj [proposed][zmn ff] an alternative way to
+  encapsulated pending LN payments ([HTLCs][topic htlc]) that would
+  reduce the amount of network bandwidth and latency required to accept
+  a payment.  More recently, Lloyd Fournier suggested the idea could
```

Comment:
Is there a good link for "suggested" text? (or move one of the follow links up here for context)

Reply:
I asked one of the discussion participants if there was a public link; if I get an answer and if there is, I'll update; otherwise, I suggest we don't add a link here to avoid conflating that possibly-private discussion with the later public mailing list discussion.